From fb13e36d6cf9e8e00974c3e3f674ff738f279024 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jan=20Arve=20S=C3=A6ther?= <jan-arve.saether@qt.io>
Date: Tue, 16 Sep 2025 14:19:33 +0200
Subject: [PATCH 056/239] Assess security levels for files under
 qtdeclarative/src/qmltest

No critical security levels found.

QUIP: 23
Fixes: QTBUG-136206
Pick-to: 6.9 6.8
Change-Id: I1a29dab77678a45cd5fe7391124791cbf6aa6ca8
Reviewed-by: Mitch Curtis <mitch.curtis@qt.io>
(cherry picked from commit 249e7f768fb7b4e11c90d2ece022738dd1d36b02)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 src/qmltest/SignalSpy.qml                   | 1 +
 src/qmltest/TestCase.qml                    | 1 +
 src/qmltest/TestSchedule.qml                | 1 +
 src/qmltest/android/androidtestutils_p.h    | 1 +
 src/qmltest/android/quickforandroidtest.cpp | 1 +
 src/qmltest/android/quickforandroidtest_p.h | 1 +
 src/qmltest/qtestoptions_p.h                | 1 +
 src/qmltest/quicktest.cpp                   | 1 +
 src/qmltest/quicktest.h                     | 1 +
 src/qmltest/quicktest_p.h                   | 1 +
 src/qmltest/quicktestevent.cpp              | 1 +
 src/qmltest/quicktestevent_p.h              | 1 +
 src/qmltest/quicktestglobal.h               | 1 +
 src/qmltest/quicktestglobal_p.h             | 1 +
 src/qmltest/quicktestresult.cpp             | 1 +
 src/qmltest/quicktestresult_p.h             | 1 +
 src/qmltest/quicktestutil.cpp               | 1 +
 src/qmltest/quicktestutil_p.h               | 1 +
 src/qmltest/testlogger.js                   | 1 +
 19 files changed, 19 insertions(+)

diff --git a/src/qmltest/SignalSpy.qml b/src/qmltest/SignalSpy.qml
index be67a68df1..eb23a52509 100644
--- a/src/qmltest/SignalSpy.qml
+++ b/src/qmltest/SignalSpy.qml
@@ -1,5 +1,6 @@
 // Copyright (C) 2021 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 import QtQuick 2.0
 import QtTest 1.1
diff --git a/src/qmltest/TestCase.qml b/src/qmltest/TestCase.qml
index 8c3383e21d..76364fe4d4 100644
--- a/src/qmltest/TestCase.qml
+++ b/src/qmltest/TestCase.qml
@@ -1,5 +1,6 @@
 // Copyright (C) 2016 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 import QtQuick 2.0
 import QtQuick.Window 2.0 // used for qtest_verifyItem
diff --git a/src/qmltest/TestSchedule.qml b/src/qmltest/TestSchedule.qml
index a2d291f80d..2dbceb8a1c 100644
--- a/src/qmltest/TestSchedule.qml
+++ b/src/qmltest/TestSchedule.qml
@@ -1,5 +1,6 @@
 // Copyright (C) 2022 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 pragma Singleton
 import QtQml
diff --git a/src/qmltest/android/androidtestutils_p.h b/src/qmltest/android/androidtestutils_p.h
index 6acaa1cf4e..5c827246b6 100644
--- a/src/qmltest/android/androidtestutils_p.h
+++ b/src/qmltest/android/androidtestutils_p.h
@@ -1,5 +1,6 @@
 // Copyright (C) 2025 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #ifndef ANDROIDTESTUTILS_P_H
 #define ANDROIDTESTUTILS_P_H
diff --git a/src/qmltest/android/quickforandroidtest.cpp b/src/qmltest/android/quickforandroidtest.cpp
index 642e6539c4..ec561872eb 100644
--- a/src/qmltest/android/quickforandroidtest.cpp
+++ b/src/qmltest/android/quickforandroidtest.cpp
@@ -1,5 +1,6 @@
 // Copyright (C) 2025 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #include "quickforandroidtest_p.h"
 #include "androidtestutils_p.h"
diff --git a/src/qmltest/android/quickforandroidtest_p.h b/src/qmltest/android/quickforandroidtest_p.h
index 8ae102a724..c3da60cc6a 100644
--- a/src/qmltest/android/quickforandroidtest_p.h
+++ b/src/qmltest/android/quickforandroidtest_p.h
@@ -1,5 +1,6 @@
 // Copyright (C) 2025 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #ifndef QUICKFORANDROIDTEST_P_H
 #define QUICKFORANDROIDTEST_P_H
diff --git a/src/qmltest/qtestoptions_p.h b/src/qmltest/qtestoptions_p.h
index 3e2d01a84a..d95192e38e 100644
--- a/src/qmltest/qtestoptions_p.h
+++ b/src/qmltest/qtestoptions_p.h
@@ -1,5 +1,6 @@
 // Copyright (C) 2018 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #ifndef QTESTOPTIONS_P_H
 #define QTESTOPTIONS_P_H
diff --git a/src/qmltest/quicktest.cpp b/src/qmltest/quicktest.cpp
index dbfac1d9ea..511342d3ff 100644
--- a/src/qmltest/quicktest.cpp
+++ b/src/qmltest/quicktest.cpp
@@ -1,5 +1,6 @@
 // Copyright (C) 2016 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #include "quicktest_p.h"
 #include "quicktestresult_p.h"
diff --git a/src/qmltest/quicktest.h b/src/qmltest/quicktest.h
index 90ed3a973b..9dd393a95c 100644
--- a/src/qmltest/quicktest.h
+++ b/src/qmltest/quicktest.h
@@ -1,5 +1,6 @@
 // Copyright (C) 2016 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #ifndef QUICKTEST_H
 #define QUICKTEST_H
diff --git a/src/qmltest/quicktest_p.h b/src/qmltest/quicktest_p.h
index 2deffab8ca..851117db1c 100644
--- a/src/qmltest/quicktest_p.h
+++ b/src/qmltest/quicktest_p.h
@@ -1,5 +1,6 @@
 // Copyright (C) 2020 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #ifndef QUICKTEST_P_H
 #define QUICKTEST_P_H
diff --git a/src/qmltest/quicktestevent.cpp b/src/qmltest/quicktestevent.cpp
index 9bd2a9887c..f2eefee6fd 100644
--- a/src/qmltest/quicktestevent.cpp
+++ b/src/qmltest/quicktestevent.cpp
@@ -1,5 +1,6 @@
 // Copyright (C) 2021 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #include "quicktestevent_p.h"
 #include <QtTest/qtestkeyboard.h>
diff --git a/src/qmltest/quicktestevent_p.h b/src/qmltest/quicktestevent_p.h
index 6f73130b5b..f9be9cc421 100644
--- a/src/qmltest/quicktestevent_p.h
+++ b/src/qmltest/quicktestevent_p.h
@@ -1,5 +1,6 @@
 // Copyright (C) 2016 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #ifndef QUICKTESTEVENT_P_H
 #define QUICKTESTEVENT_P_H
diff --git a/src/qmltest/quicktestglobal.h b/src/qmltest/quicktestglobal.h
index 3f8c5c3582..f2e930d735 100644
--- a/src/qmltest/quicktestglobal.h
+++ b/src/qmltest/quicktestglobal.h
@@ -1,5 +1,6 @@
 // Copyright (C) 2016 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #ifndef TESTQUICKGLOBAL_H
 #define TESTQUICKGLOBAL_H
diff --git a/src/qmltest/quicktestglobal_p.h b/src/qmltest/quicktestglobal_p.h
index e3be0ebde1..b983e2db1e 100644
--- a/src/qmltest/quicktestglobal_p.h
+++ b/src/qmltest/quicktestglobal_p.h
@@ -1,5 +1,6 @@
 // Copyright (C) 2021 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #ifndef QUICKTESTGLOBAL_P_H
 #define QUICKTESTGLOBAL_P_H
diff --git a/src/qmltest/quicktestresult.cpp b/src/qmltest/quicktestresult.cpp
index c5014f66d4..e9cc03da87 100644
--- a/src/qmltest/quicktestresult.cpp
+++ b/src/qmltest/quicktestresult.cpp
@@ -1,6 +1,7 @@
 // Copyright (C) 2017 Crimson AS <info@crimson.no>
 // Copyright (C) 2016 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #include "quicktestresult_p.h"
 #include "quicktest.h"
diff --git a/src/qmltest/quicktestresult_p.h b/src/qmltest/quicktestresult_p.h
index 1e274e1804..d77fdfaad9 100644
--- a/src/qmltest/quicktestresult_p.h
+++ b/src/qmltest/quicktestresult_p.h
@@ -1,5 +1,6 @@
 // Copyright (C) 2016 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #ifndef QUICKTESTRESULT_P_H
 #define QUICKTESTRESULT_P_H
diff --git a/src/qmltest/quicktestutil.cpp b/src/qmltest/quicktestutil.cpp
index c7cac4b4b2..e1cc5ca3fc 100644
--- a/src/qmltest/quicktestutil.cpp
+++ b/src/qmltest/quicktestutil.cpp
@@ -1,5 +1,6 @@
 // Copyright (C) 2019 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #include "quicktestutil_p.h"
 
diff --git a/src/qmltest/quicktestutil_p.h b/src/qmltest/quicktestutil_p.h
index d9c0241cbe..767b702d8e 100644
--- a/src/qmltest/quicktestutil_p.h
+++ b/src/qmltest/quicktestutil_p.h
@@ -1,5 +1,6 @@
 // Copyright (C) 2019 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #ifndef QUICKTESTUTIL_P_H
 #define QUICKTESTUTIL_P_H
diff --git a/src/qmltest/testlogger.js b/src/qmltest/testlogger.js
index dc28bf6c73..60153f6880 100644
--- a/src/qmltest/testlogger.js
+++ b/src/qmltest/testlogger.js
@@ -1,5 +1,6 @@
 // Copyright (C) 2016 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 .pragma library
 
-- 
2.51.2

